core/src/main/java/io/fabric8/maven/core/util/Constants.java

```diff
@@ -22,4 +22,6 @@
 public class Constants {
     public static final String RESOURCE_SOURCE_URL_ANNOTATION = "maven.fabric8.io/source-url";
     public static final String RESOURCE_APP_CATALOG_ANNOTATION = "maven.fabric8.io/app-catalog";
+    public static final String SPRING_BOOT_MAVEN_PLUGIN_GA = "org.springframework.boot:spring-boot-maven-plugin";
```

Comment:
Don't think its a good idea to add tech specific constants in the core package. said that several times, but I guess I'm wrong somehow. would be happy to be enlighted (btw, we are currently on the way to exactly do the reverse, to decouple stuff. See the e.g. `fabric8:watch` refactorings, where we move out spring specific stuff from the core.